lib/linux/background-managers/manager.js.template

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+exports.available = async () => {
+	return false;
+};
+
+exports.set = async (imagePath) => {
+	return false;
+};
+
+exports.get = async () => {
+	return "";
+};
```

Comment:
I don't think this file is needed. It's just as easy to just copy-paste from an existing manager file.